docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.adoc

```diff
@@ -47,3 +47,29 @@ com.company.certificate.MyCertificateValidator
 ----
 
 NOTE: You can @Inject CDI and EJB beans in your validator implementation, however since the validator itself is not a Jakarta EE bean, `@PostConstruct` and `@PreDestroy`-annotated methods will not be called on the validator instance.
+
+[[client-certificate-expiration-validator]]
+== Certificate Expiration Validation
+By default, certificate expiration validation is enabled. This validation checks if the certificates in the trust store are valid time wise. If the client certificate has expired or is not valid yet, a warning will be printed in the server logs.
+
+=== Using the Web Admin Console
+
+This feature can be configured on the default certificate realm as follows on the Web administration console:
+
+. Navigate to the applicable configuration page for your use (e.g. `server-config`) under the `Configurations` option
+in the side menu
+. Head to `Security` -> `Realms` and select the `certificate` realm
+. Set the property `certificate-validation` _value_ to `[true/false]`
+
+image:security/default-certificate-expiration-validation.png[Certificate Expiration Validation Setting]
+
+=== Using an `asadmin` command
```

Comment:
These are backwards I think.
See the style guide: https://payara.atlassian.net/wiki/spaces/SUP/pages/2149515318/Payara+Documentation+Style+Guide#Configuration---Top-Level-Header-(If-Applicable)